pkg/services/app/remove.go

```diff
@@ -0,0 +1,56 @@
+package app
+
+import (
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	wego "github.com/weaveworks/weave-gitops/api/v1alpha1"
+)
+
+type AutomationManifestPaths struct { // source for automation isn't currently stored
+	AppDirAutomationPath    string
+	TargetDirAutomationPath string
+}
+
+func dirExists(d string) bool {
```

Comment:
small nit: this function already exists in utils, works for both files and directories

Reply:
Cool. Thanks for pointing that out!

Reply:
Actually, after looking... I need to know it's a directory :man_shrugging: